bin/fix/fix-audit.rb

```diff
@@ -0,0 +1,188 @@
+require 'rest-client'
+# we use active_fedora/noid to generate pairtrees from noids
+require 'active_fedora/noid'
+require 'yaml'
+require 'rdf/turtle'
+require File.expand_path('../../../config/environment', __FILE__)
+
+def read_config
+  rails_env = Rails.env
+  config = YAML.load_file("config/fedora.yml")
+  @user = config[rails_env]['user']
+  @password = config[rails_env]['password']
+  @rest = config[rails_env]['url']
+  @base_path = config[rails_env]['base_path']
+end
+
+def patch(path, payload)
+  puts 'Patching ' + path + ': ' + payload
```

Comment:
@pgwillia since this is a raw script with no active Rails environment or Rakefile logger available, I'm curious what the value of that would be versus just tee'ing stdout into a file when this script is run?